### PR TITLE
Add support for default visible dimensions and measures in dashboard config

### PR DESCRIPTION
**Pull request by AdityaHegde**
Addressing [Support default selected dimensions and measures](https://www.notion.so/rilldata/Setting-defaults-for-selected-dimensions-measures-a542c3abf325450e896dc411f1167c91)

- Adds parser support for `default_measures` and `default_dimensions` in dashboard config
- Adds selecting of measures and dimensions before creating a dashboard store.

### CODE REVIEW

File: proto/rill/runtime/v1/resources.proto [hunk 1]
```diff
@@ -185,8 +185,10 @@ message MetricsViewSpec {
   string time_dimension = 5;
   // Dimensions in the metrics view
   repeated DimensionV2 dimensions = 6;
+  repeated string default_selected_dimensions = 18;
   // Measures in the metrics view
   repeated MeasureV2 measures = 7;
+  repeated string default_selected_measures = 19;
```
[the remark this answers is not in the document]
Comment (AdityaHegde) [reply proto/rill/runtime/v1/resources.proto:191]: Either is fine with me. I just kept the same field as product requested.

File: runtime/compilers/rillv1/parse_metrics_view.go [hunk 2]
```diff
@@ -425,6 +445,7 @@ func (p *Parser) parseMetricsView(ctx context.Context, node *Node) error {
 			Unnest:      dim.Unnest,
 		})
 	}
+	spec.DefaultSelectedDimensions = slices.Clone(tmp.DefaultSelectedDimensions)
```
[the remark this answers is not in the document]
Comment (AdityaHegde) [reply runtime/compilers/rillv1/parse_metrics_view.go:448]: We dont need it.

File: proto/rill/runtime/v1/resources.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdityaHegde) [reply proto/rill/runtime/v1/resources.proto]: Changed after consulting product.

**Review by begelundmuller — APPROVED** [review]
LGTM

**Pull request merged**